CONVERT_SERVICE/convert_service/convert_image/model/convert_image.py

```diff
@@ -9,23 +9,44 @@
 # accordance with the terms of the license agreement you entered into
 # with Jalasoft.
 #
-
+from django.contrib.sites.shortcuts import get_current_site
+from django.http import HttpResponse
 from wand.image import Image
-from wand.display import display
 from wand.color import Color
 import datetime
+import json
 
 
 class ConvertImage:
     """Class to convert image with many params"""
```

Comment:
Please may you clarify the class description?... be more specific.

Reply:
resolved